gui/src/components/Onboarding/Steps.tsx

```diff
@@ -1,5 +1,20 @@
-import { Link, Stack, TextField, Typography } from "@mui/material";
-import { ChangeEvent } from "react";
+import {
+  Alert,
+  Checkbox,
+  Collapse,
+  FormControlLabel,
+  Grid,
+  Link,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import { invoke } from "@tauri-apps/api/tauri";
+import { ChangeEvent, useState } from "react";
+import { type FieldValues } from "react-hook-form"; // TODO: check this import
```

Comment:
what's the problem with this import?

Reply:
Forgot to delete this comment 😅 
But this was due to the signature on the `HDWalletForm` `onSubmit` prop being `Props.onSubmit: (data: FormType) => void` and I was passing it this `onSubmit: (params: FieldValues) => void`